src/components/HeroHeader/SettingsButton.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Query, Q, models } from 'cozy-client'
+import AppLinker from 'cozy-ui/react/AppLinker'
+import { translate } from 'cozy-ui/react/I18n'
+import get from 'lodash/get'
+
+const { applications } = models
+
+import CornerButton from './CornerButton'
+
+const SettingsButton = ({ lang }) => (
+  <Query query={() => Q('io.cozy.apps').where({ slug: 'settings' })}>
```

Comment:
can we use a queryConnect instead? In order to name the query? Also having a fetchPolicy cacheFirst or X minutes can be a good improvement, no?

Reply:
Oh right I wanted to add the fetchPolicy and forgot, thanks! And yeah i'll use queryConnect instead 👍